src/Cloud/DecoratorPluginManager.php

```diff
@@ -21,38 +21,16 @@
  */
 class DecoratorPluginManager extends AbstractPluginManager
 {
-    /**
-     * Default set of decorators
-     *
-     * @var array
-     */
-    protected $invokableClasses = [
-        'htmlcloud' => 'Zend\Tag\Cloud\Decorator\HtmlCloud',
-        'htmltag'   => 'Zend\Tag\Cloud\Decorator\HtmlTag',
-        'tag'       => 'Zend\Tag\Cloud\Decorator\HtmlTag',
+    protected $aliases = [
+        'htmlcloud' => Decorator\HtmlCloud::class,
+        'htmltag'   => Decorator\HtmlTag::class,
```

Comment:
The above two should also have the aliases `htmlCloud` and 'htmlTag`, as those were used previously. I can do that on merge, though.